components/tag/src/tag-add/tag-add.scss

```diff
@@ -62,12 +74,12 @@
   width: 16px;
   height: 16px;
   vertical-align: text-top;
-  display: inline-block;
   fill: #ffffff;
 }
 
 .dt-tag-add-icon-button {
   fill: $turquoise-600;
+  padding-top: -2px;
```

Comment:
minus padding does not work. This is an invalid value. @tomheller can we add a stylelint rule for such values?